### PR TITLE
LTC 5 Cards

### PR DESCRIPTION
**Pull request by squee1968**

### CODE REVIEW

**Comment by squee1968** [comment]
Can't seem to delete the Binary files

**Comment by squee1968** [comment]
Thankee-sai, Kevlahnota for the assist

File: forge-gui/res/cardsfolder/upcoming/gimli_of_the_glittering_caves.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Types:Legendary Creature Dwarf Warrior
+PT:1/1
+K:Double strike
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.Legendary+Other+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever another legendary creature enters the battlefield under your control, put a +1/+1 counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigTreasure | TriggerDescription$ Whenever NICKNAME deals combat damage to a player, create a Treasure token.
+SVar:TrigTreasure:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_treasure_sac | TokenOwner$ You
+SVar:BuffedBy:Creature.Legendary
+DeckHas:Ability$Token
+Oracle:First strike\nWhenever another legendary creature enters the battlefield under your control, put a +1/+1 counter on Gimli of the Glittering Caves.\nWhenever Gimli deals combat damage to a player, create a Treasure token.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/gimli_of_the_glittering_caves.txt:12]: wrong KW?

File: forge-gui/res/cardsfolder/upcoming/legolas_greenleaf.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Types:Legendary Creature Elf Archer
+PT:2/2
+K:Reach
+S:Mode$ CantBlockBy | ValidAttacker$ Creature.Self | ValidBlocker$ Creature.powerLE2 | Description$ CARDNAME can't be blocked by creatures with power 2 or less.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.Legendary+Other+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever another legendary creature enters the battlefield under your control, put a +1/+1 counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigDraw | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, draw a card.
+SVar:TrigDraw:DB$ Draw
+SVar:BuffedBy:Creature.Legendary
+DeckHas:Ability$Token
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/legolas_greenleaf.txt:12]: I don't see token

File: forge-gui/res/cardsfolder/upcoming/merry_warden_of_isengard.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Merry, Warden of Isengard
+ManaCost:1 G W
+Types:Legendary Creature Halfling Advisor
+PT:1/4
+K:Partner:Pippin, Warden of Isengard:Pippin
+T:Mode$ ChangesZoneAll | ValidCards$ Artifact.YouCtrl | Destination$ Battlefield | TriggerZones$ Battlefield | ActivationLimit$ 1 | Execute$ TrigToken | TriggerDescription$ Whenever one or more artifacts enter the battlefield under your control, create a 1/1 white Soldier creature token with lifelink. This ability triggers only once each turn.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ w_1_1_soldier_lifelink | TokenOwner$ You
+DeckHints:Name$Merry, Warden of Isengard
+DeckHas:Ability$Token|Counters
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/merry_warden_of_isengard.txt:9]: I don't see counters

File: forge-gui/res/cardsfolder/upcoming/merry_warden_of_isengard.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Name:Merry, Warden of Isengard
+ManaCost:1 G W
+Types:Legendary Creature Halfling Advisor
+PT:1/4
+K:Partner:Pippin, Warden of Isengard:Pippin
+T:Mode$ ChangesZoneAll | ValidCards$ Artifact.YouCtrl | Destination$ Battlefield | TriggerZones$ Battlefield | ActivationLimit$ 1 | Execute$ TrigToken | TriggerDescription$ Whenever one or more artifacts enter the battlefield under your control, create a 1/1 white Soldier creature token with lifelink. This ability triggers only once each turn.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ w_1_1_soldier_lifelink | TokenOwner$ You
+DeckHints:Name$Merry, Warden of Isengard
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/merry_warden_of_isengard.txt:8]: Doesn't make sense to request itself

File: forge-gui/res/cardsfolder/upcoming/treebeard,_gracious_host.txt [hunk 5]
```diff
@@ -0,0 +1,14 @@
+Name:Treebeard, Gracious Host
+ManaCost:2 G W
+Types:Legendary Creature Treefolk
+PT:0/5
+K:Trample,
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/treebeard,_gracious_host.txt:5]: remove ,

File: forge-gui/res/cardsfolder/upcoming/legolas_greenleaf.txt [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+Types:Legendary Creature Elf Archer
+PT:2/2
+K:Reach
+S:Mode$ CantBlockBy | ValidAttacker$ Creature.Self | ValidBlocker$ Creature.powerLE2 | Description$ CARDNAME can't be blocked by creatures with power 2 or less.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.Legendary+Other+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever another legendary creature enters the battlefield under your control, put a +1/+1 counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigDraw | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, draw a card.
+SVar:TrigDraw:DB$ Draw
+SVar:BuffedBy:Creature.Legendary
+DeckHas:Ability$Counter
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/legolas_greenleaf.txt:12]: "Counter" is incomplete

File: forge-gui/res/cardsfolder/upcoming/treebeard,_gracious_host.txt
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/treebeard,_gracious_host.txt:1]: invalid filename

**Review by tool4ever — APPROVED** [review]
thanks

**Pull request merged**